docs/classical_control.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<p>Add that <code>CircuitOperation</code> is the only way to add scope that isn't global to the whole circuit, and what to avoid naming your keys to not interfere with the scope-renaming (if possible).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5403/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Added the note about scope. Scope-renaming of keys is protected in Cirq (the separator `:` is not permitted in user-specified keys), so no special caution is necessary.